application/src/main/java/org/togetherjava/tjbot/commands/generic/PingCommand.java

```diff
@@ -0,0 +1,35 @@
+package org.togetherjava.tjbot.commands.generic;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.togetherjava.tjbot.commands.example.AbstractCommand;
+
+/**
+ * Ping Pong command.
+ */
+public class PingCommand extends AbstractCommand {
+    private static final Logger logger = LoggerFactory.getLogger(PingCommand.class);
+
+    public PingCommand() {
+        super("Ping Pong",
+                "This command returns a message with the value `pong`, to show the user that the bot is up and running.",
+                false);
+    }
+
+    /**
+     * When the slash command is `/ping`, then the bot returns with the value `Pong!`
+     * 
+     * @param event The relating {@link SlashCommandEvent}
+     */
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        if (!"/ping".equals(event.getCommandString())) {
```

Comment:
The commandhandler automatically sorts this out for you, you only receive commands when it's _your_ command

Reply:
I prefer to have a defensive code strategy in here despite of that

Reply:
I don't like the idea of useless if statements, you aren't even checking or it's the right command correctly.